sync/syncer.go

```diff
@@ -485,27 +485,41 @@ func (s *Syncer) DataAvailabilty(blk *types.Block) ([]*types.AddressableSignedTr
 	return txs, atxs, nil
 }
 
-func (s *Syncer) fetchLayerBlockIds(m map[string]p2p.Peer, lyr types.LayerID) ([]types.BlockID, error) {
+func (s *Syncer) fetchLayerBlockIds(m map[uint32][]p2p.Peer, lyr types.LayerID) ([]types.BlockID, error) {
```

Comment:
Why do we return a slice of blockIds? Can't we return the map that is used inside this func instead?

Reply:
its more convenient this way, the next methods in the flow expect a slice

Reply:
I read the next methods "GetFullBlocks" and "blockSlliceTocChan" - don't see a reason to keep it this way.
Consider opening a technical debt since this results in duplication and extra memory usage which seem unnecessary at the moment.